ros_ethercat_model/include/ros_ethercat_model/ros_ethercat.hpp

```diff
@@ -171,24 +178,34 @@ class RosEthercat : public hardware_interface::RobotHW
     std::string robot_description_param;
     bool allow;
 
+    robot_state_name_ = robot_hw_nh.getNamespace().substr(1); // remove the leading slash of the namespace
+    ROS_INFO_STREAM("Robot State Name: " << robot_state_name_);
+
     if (!robot_hw_nh.getParam("robot_description_param", robot_description_param))
     {
       ROS_ERROR("robot_description_param not found (namespace: %s)", robot_hw_nh.getNamespace().c_str());
       return false;
     }
 
-    if (!root_nh.getParam(robot_description_param, robot_description))
+    if (robot_description_param == "none")
```

Comment:
This is to allow to have ronex modules "alone" in a RosEthercat without having to define a URDF.
None could be fine. This will be a parameter in a yaml file.

```yaml
robot_description_param: None
```